RequestManagementSystem/Client/Request.py

```diff
@@ -370,7 +370,7 @@ def Status( self, value ):
     # If the status moved to Failed or Done, update the lastUpdate time
     if value in ( 'Done', 'Failed' ):
       if value != self._Status:
-        self.LastUpdate = datetime.datetime.utcnow().replace( microsecond = 0 )
```

Comment:
What do you think about adding
```
# pylint: disable=E0202 
```
to the respective lines to hide the pylint error message?